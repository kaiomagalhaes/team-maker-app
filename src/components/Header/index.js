import Link from 'next/link';
import NProgress from 'nprogress';
import Router from 'next/router';

import { APP_NAME } from 'config';

Router.onRouteChangeStart = () => {
  NProgress.start();
};
Router.onRouteChangeComplete = () => {
  NProgress.done();
};

Router.onRouteChangeError = () => {
  NProgress.done();
};

const Header = () => (
  <div>
    <Link href="/">
      <a>{APP_NAME}</a>
    </Link>
  </div>
);

export default Header;
