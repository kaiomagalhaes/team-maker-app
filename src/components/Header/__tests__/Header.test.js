import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import Header from '../index';

describe('Header', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<Header />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
