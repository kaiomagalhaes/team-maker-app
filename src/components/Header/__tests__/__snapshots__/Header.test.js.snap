// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders and matches the snapshot 1`] = `
<div>
  <Link
    href="/"
  >
    <a>
      APP NAME
    </a>
  </Link>
</div>
`;
